Source/d_loop.c

```diff
@@ -718,7 +718,20 @@ void TryRunTics (void)
 
     if (new_sync)
     {
-	counts = availabletics;
+        if (uncapped)
```

Comment:
I haven't received any feedback and it looks like it will take a while. We can add this check for safety, currently uncapped multiplayer does not work anyway, so we will not make it worse.